content/docs/dvclive/api-reference/live/log_param.md

```diff
@@ -24,12 +22,22 @@ live.log_param("n_iter", 50)
 On each `live.log_param(name, val)` call, DVCLive will write the `name`
 parameter to `{Live.dir}/params.yaml` with the corresponding `val`:
 
-```dvc
+```cli
 $ cat dvclive/params.yaml
 lr: 1e-3
 n_iter: 50
 ```
```

Comment:
Minor again but I incline to use YAML blocks for these, e.g.:

````suggestion
```yaml
lr: 1e-3
n_iter: 50
```
````

You'd just have to make sure it's mentioned shortly after that we're looking at dvclive/params.yaml .

Reply:
I mean shortly before.